localization/ndt_scan_matcher/include/ndt_scan_matcher/hyper_parameters.hpp

```diff
@@ -81,7 +87,10 @@ struct HyperParameters
     struct CovarianceEstimation
     {
       bool enable{};
+      CovarianceEstimationType covariance_estimation_type{};
       std::vector<Eigen::Vector2d> initial_pose_offset_model{};
+      std::vector<double> initial_pose_offset_model_x{};
+      std::vector<double> initial_pose_offset_model_y{};
```

Comment:
`initial_pose_offset_model` and `*_model_x` & `*_model_y` seem to have the same values. Since this is redundant, can't you simplify it?

Reply:
`initial_pose_offset_model_x` and `initial_pose_offset_model_y` are necessary to use `pclomp::propose_poses_to_search()`.

On the other hand, `initial_pose_offset_model` is not used as is by `ndt_scan_matcher`.

Therefore, I am  considering deleting `std::vector<Eigen::Vector2d> initial_pose_offset_model` from hyper_parameters.hpp.

Reply:
Your suggestion seems good! :+1: